src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -564,14 +565,36 @@ public static void assertFileIsReadable(final File file) {
     public static void assertFileIsReadable(final Path path) {
         Utils.nonNull(path);
 
-        if ( ! Files.exists(path) ) {
-            throw new UserException.CouldNotReadInputFile(path, "It doesn't exist.");
-        }
-        if ( ! Files.isRegularFile(path) ) {
-            throw new UserException.CouldNotReadInputFile(path, "It isn't a regular file");
-        }
-        if ( ! Files.isReadable(path) ) {
-            throw new UserException.CouldNotReadInputFile(path, "It is not readable, check the file permissions");
+        try {
```

Comment:
@jean-philippe-martin It's a good idea to translate this into a clearer exception message.  I think most of the logging is unnecessary because you can already get the stack trace, which I'm assuming will include the code, location, reason, and message in it?  

Do you think we need a top-level catch for StorageException in Main in order to translate them if their thrown in other places?  Does the storage exception include the information about the path that was being accessed?  I assume that's very similar information to the location that it includes. ?